src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -173,7 +189,10 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
                     autoReferenced = true;
                 }
 
-                _versions.Add(new DisplayVersion(version.version, additionalInfo: string.Empty, isCurrentInstalled: installed, autoReferenced: autoReferenced, isDeprecated: version.isDeprecated));
+                if (!installed)
+                {
+                    _versions.Add(new DisplayVersion(version.version, additionalInfo: string.Empty, isCurrentInstalled: installed, autoReferenced: autoReferenced, isDeprecated: version.isDeprecated));
```

Comment:
What is the difference between passing string.Empty for additionalInfo here versus passing null? We pass null in a few other places so I'm wondering how that affects the DisplayVersion. It's not quite clear from quickly looking at the DisplayVersion. Thanks!

Reply:
There is no difference, `AditionalInfo` is used for _toString property, and there we check with `string.IsNullOrEmpty` even though I think is better if we use string.Empty so there is no confusion that AdditionalInfo is a string

Reply:
I think it's actually better to have it as null in all places if it's treated the same.